ngraph/core/include/ngraph/ops.hpp

```diff
@@ -170,3 +170,5 @@
 #include "ngraph/op/util/op_types.hpp"
 #include "ngraph/op/variadic_split.hpp"
 #include "ngraph/op/xor.hpp"
+#include "openvino/op/nv12_to_bgr.hpp"
+#include "openvino/op/nv12_to_rgb.hpp"
```

Comment:
I believe we cannot use `ov` directly in ngraph. We probably need to create a ngraph duplicate headers for these operations

CC @ilyachur

Reply:
We already have a new header:
https://github.com/openvinotoolkit/openvino/blob/master/ngraph/core/include/openvino/op/ops.hpp

Reply:
yes, we have, but new operation can be added to old header as well, because both API should work

Reply:
Sure

Reply:
fixed